app/controllers/comments-controller.js

```diff
@@ -0,0 +1,13 @@
+const {selectCommentsByArticleID} = require('../models/comments-model')
+
+exports.getCommentsByArticleID = (req, res, next)=>{
+    const articleID = req.params.article_id
+
+    selectCommentsByArticleID(articleID)
+    .then((rows)=>{
```

Comment:
Ideally we want to call this something more descriptive than rows. Something like comments would be more fitting.